projects/igniteui-angular/src/lib/query-builder/query-builder-functions.spec.ts

```diff
@@ -530,10 +530,10 @@ export class QueryBuilderFunctions {
     public static verifyEditModeQueryExpressionInputStates(fix,
         entitySelectEnabled: boolean,
         fieldComboEnabled: boolean,
-        columnSelectEnabled: boolean,
-        operatorSelectEnabled: boolean,
-        valueInputEnabled: boolean,
-        commitButtonEnabled: boolean,
+        columnSelectEnabled: boolean = null,
```

Comment:
Why not just make these params optional instead of initializing with null?

Reply:
Agree. Syntax changed